integration-tests/gradle/src/main/resources/spring-dependency-plugin-project/build.gradle

```diff
@@ -0,0 +1,46 @@
+plugins {
+    id 'java'
+    id 'io.quarkus'
+    id 'application'
```

Comment:
I think we can remove this plugin and its configuration as it's not used in the test

Reply:
Remoed